files/scripts/benchmark_job.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -ex
+
+helpmsg="benchmark_job.sh --workspace [WORKSPACE] --node [node] --compiler [compiler] --build-number [build_number] --git-branch [harness_gitbranch] --benchmark [benchmark_name] --compiler_flags compiler_flags --linker_flags linker_flags --harness_options harness_options --run_flags run_flags --iterations iterations --size size"
+
+ARGUMENT_LIST=(
+	"workspace"
+	"node"
+	"compiler"
+	"build_number"
+	"git_branch"
+	"benchmark"
+	"compiler_flags"
+	"linker_flags"
+	"harness_options"
+	"iterations"
+	"size"
+	"run_flags"
+)
+
+root_dir="$(realpath $(dirname $0))"
+
+. ${root_dir}/argparse.sh
+
+if [ ! -n ${workspace} ] || [ ! -n ${node} ] || [ ! -n ${compiler} ] || [ ! -n ${build_number} ] || [ ! -n ${git_branch} ] || [ ! -n ${benchmark} ]; then
+	echo "Missing Required Argument(s) !!!"
```

Comment:
No need for exclamation marks... :)

Reply:
It does make it standout in the midst of the set -ex very verbose output.
I added some at the start of the line, to make it even more stand out :)